functions/reporting/reporting.py

```diff
@@ -23,7 +23,7 @@ def current_status_report(self, device_group_config, updated):
             "current_device_group_config": device_group_config,
             "device_group": self.device_group,
             "report_creation_time": int(time.time()),
-            "hostname": get_fqdn(),
+            "hostname": get_ip_address_from_env(),
```

Comment:
The hostname should resolve to `get_fqdn(),`  please add another field like `ip_address` to resolve to the ip address that's passed from the envvar (or the sane default of None/etc)